### PR TITLE
#26 revisited

### PR DESCRIPTION
**Pull request by Brian-Drummond**
Filename and line number added to error report, following Tristan's suggestion.

### CODE REVIEW

**Comment by tgingold** [comment]
I think you should rebase your branch, otherwise the continuous builders fail.

File: src/grt/grt-errors.adb [hunk 1]
```diff
@@ -254,6 +254,20 @@ package body Grt.Errors is
...
+                    Filename : Ghdl_C_String;
+                    Line : Ghdl_I32) is
+   begin
+      Error_H;
+      Put_Err (Str);
+      Put_Err (" at ");
+      Put_Err (Filename);
+      Put_Err (" line ");
+      Put_I32 (Error_Stream, Line);
+      Newline_Err;
```
Comment (tgingold) [inline src/grt/grt-errors.adb]: grt.lib.ghdl_program_error only uses Error_* subprograms, and I think this is correct.
Might be worth creating a subprogram for filename + line.

Comment (Brian-Drummond) [reply src/grt/grt-errors.adb]: OK thanks, please ignore for now (I need to figure out rebase!)

Comment (Brian-Drummond) [reply src/grt/grt-errors.adb]: Tried rebasing, we'll see...
I thought exporting an Error subprogram that understood location could be useful for other runtime errors ... (note the `Put_Err` calls in the snippet above are only used in grt_errors). And some other subprograms in grt.lib use Error not Error_*, ... or I may be misunderstanding your point?

**Pull request closed without merging**